src/proto/message.proto

```diff
@@ -210,4 +213,16 @@ message FrameActionBody {
   bytes state = 5; // Serialized frame state value
   bytes transaction_id = 6; // Chain-specific transaction ID for tx actions
   bytes address = 7; // Chain-specific address for tx actions
+}
+
+enum StorageUnitType {
+  UNIT_TYPE_LEGACY = 0;
+  UNIT_TYPE_2024 = 1;
+  UNIT_TYPE_2025 = 2;
+}
+
+message LendStorageBody {
+  uint64 to_fid = 1;
```

Comment:
We don't need the from fid? Might be useful to have it just in case, esp.  if this is the same body that will be emitted as an event. I can see wanting to know who the storage came from.